.travis.yml

```diff
@@ -19,6 +19,8 @@ env:
   - PYTHON_VERSION=3.6 TESTCMD=" -vv --with-timer --with-coverage --cover-package=pymc3 -e test_examples -e test_distributions"
   - PYTHON_VERSION=3.6 TESTCMD=" -vv --with-timer --with-coverage --cover-package=pymc3 pymc3.tests.test_distributions pymc3.tests.test_examples"
   - PYTHON_VERSION=3.6 TESTCMD=" -vv --with-timer --with-coverage --cover-package=pymc3 pymc3.tests.test_distributions_random"
+  - THEANO_FLAGS='floatX=float32'
```

Comment:
Would the variable be some thing like this- `THEANO_FLOAT_TYPE="floatX=float32"`?